packages/roosterjs-content-model/lib/modelApi/block/setModelAlignment.ts

```diff
@@ -0,0 +1,80 @@
+import { ContentModelDocument } from '../../publicTypes/group/ContentModelDocument';
+import { ContentModelListItem } from '../../publicTypes/group/ContentModelListItem';
+import { ContentModelTable } from '../../publicTypes/block/ContentModelTable';
+import { getOperationalBlocks } from '../selection/collectSelections';
+import { isBlockGroupOfType } from '../common/isBlockGroupOfType';
+
+const ResultMap: Record<
+    'left' | 'center' | 'right',
+    Record<'ltr' | 'rtl', 'start' | 'center' | 'end'>
+> = {
+    left: {
+        ltr: 'start',
+        rtl: 'end',
+    },
+    center: {
+        ltr: 'center',
+        rtl: 'center',
+    },
+    right: {
+        ltr: 'end',
+        rtl: 'start',
+    },
+};
+
+/**
+ * @internal
+ */
+export function setModelAlignment(
+    model: ContentModelDocument,
+    alignment: 'left' | 'center' | 'right'
+) {
+    const paragraphOrListItemOrTable = getOperationalBlocks<ContentModelListItem>(
+        model,
+        ['ListItem'],
+        ['TableCell']
```

Comment:
Do we really need to add TableCell here?

If full table is selected, I think `getOperationalBlocks` should give you the whole table block directly.